test/controllers/beneficial.owner.type.controller.spec.ts

```diff
@@ -412,8 +412,8 @@ describe("BENEFICIAL OWNER TYPE controller", () => {
     });
 
     const expectTextToContainBOTypePageURL = (text: string) => {
-      expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(1);
-      expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(config.ACTIVE_SUBMISSION_BASE_PATH);
+      expect(mockGetUrlWithParamsToPath).toHaveBeenCalledTimes(2);
+      // expect(mockGetUrlWithParamsToPath.mock.calls[0][0]).toEqual(config.ACTIVE_SUBMISSION_BASE_PATH);
```

Comment:
Should this line be commented out? I'd expect not.

Reply:
This is now captured using the `config.BENEFICIAL_OWNER_STATEMENTS_WITH_PARAMS_URL` which includes the `ACTIVE_SUBMISSION_BASE_PATH` so will remove the commented out line